WatchParty/backend/README.md

```diff
@@ -0,0 +1,8 @@
+# WatchParty Backend API
+
+This project was bootstrapped with [express-generator](https://www.npmjs.com/package/express-generator)
+
+### Scripts for execution
+
+* npm run start - start development server
+* npm run build - start production server
```

Comment:
Nice :) We'll add test soon when we get to that story. We'll groom tomorrow our next sprint, I think we are ready to get tests going after this is merged for backend and frontend.

Reply:
Actually, can you add `test` to the script to use `jest` and have it have some dummy check, just we sandbox can deploy it (it shouldn't deploy if no tests -- I'll implement that job dependence tomorrow)